homework/index.js

```diff
@@ -16,31 +37,139 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
+  function Contributions(inText) {
+    const url = inText.innerHTML;
+    const infoContainer = document.getElementById('info_container');
+    const infoRight = createAndAppend('div', infoContainer, {
+      class: 'info_right',
+      text: 'Contributions:',
+    });
+    fetchJSON(url, (err, data) => {
```

Comment:
Avoid generic names such as `data` if a better name can be found. In this case you are fetching `contributors`.

Reply:
for now , I replaced what I can. (the new work is on the way)